src/LetterChanger.java

```diff
@@ -1,37 +1,44 @@
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Scanner;
 
 class LetterChanger {
     private static Scanner scanner = new Scanner(System.in);
-    OperationsOnWords onWords = new OperationsOnWords();
+    private OperationsOnWords operations = new OperationsOnWords();
 
-    private final String commendToUpperCase = "upper";
-    private final String commendTolowerCase = "lower";
+    private final String COMMAND_TO_UPPER_CASE = "upper";
```

Comment:
Now that we have finals in proper naming convention I think we could use easier names like - UPPER_CASE